src/uwtools/tests/test_cli.py

```diff
@@ -696,7 +696,7 @@ def test_main_fail_exception_log():
                 with raises(SystemExit) as e:
                     cli.main()
                 assert e.value.code == 1
-            assert log.called_once_with(msg)
+            log.error.assert_called_once_with(msg)
```

Comment:
Bug fixed -- the old `assert` was not doing what was intended. This is the correct syntax.